lib/evss/disability_compensation_form/form526_to_lighthouse_transform.rb

```diff
@@ -379,8 +380,40 @@ def fill_change_of_address(change_of_address_source, change_of_address)
         end
       end
 
+      # only needs currentlyHomeless from 'homelessness' source
+      def fill_currently_homeless(source, target)
+        options = source['homelessSituationType']&.strip
+        send_other_description = options == 'OTHER'
+        other_description = source['otherLivingSituation'] || nil
+        target.currently_homeless = Requests::CurrentlyHomeless.new(
+          homeless_situation_options: options,
+          other_description: send_other_description ? other_description : nil
+        )
+      end
+
+      # needs whole `homelessness` object source
+      def fill_risk_of_becoming_homeless(source, target)
+        target.risk_of_becoming_homeless = Requests::RiskOfBecomingHomeless.new(
+          living_situation_options: source['homelessnessRisk']['homelessnessRiskSituationType'],
+          other_description: source['homelessnessRisk']['otherLivingSituation']
+        )
+        target.point_of_contact = source['pointOfContact']['pointOfContactName']
+        primary_phone = source['pointOfContact']['primaryPhone']
+        target.point_of_contact_number = Requests::ContactNumber.new(
+          telephone: primary_phone['areaCode'] + primary_phone['phoneNumber']
+        )
+      end
```

Comment:
(Maybe) not relevant to your implementation, but just a note that are now optional, and will likely be updated va.gov side in the near future